### ISSUE TITLE
Missing files in sdist

### ISSUE THREAD

**Issue by thatch** [sharkbound/PythonTwitchBotFramework#20]
It appears that the manifest is missing at least one file necessary to build
from the sdist for version 1.11.4. You're in good company, about 5% of other
projects updated in the last year are also missing files.

```
+ /tmp/venv/bin/pip3 wheel --no-binary pythontwitchbotframework -w /tmp/ext pythontwitchbotframework==1.11.4
Looking in indexes: http://10.10.0.139:9191/root/pypi/+simple/
Collecting pythontwitchbotframework==1.11.4
  Downloading http://10.10.0.139:9191/root/pypi/%2Bf/f8d/907c12a15a924/PythonTwitchBotFramework-1.11.4.tar.gz (56 kB)
    ERROR: Command errored out with exit status 1:
     command: /tmp/venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-wheel-oei7st_8/pythontwitchbotframework/setup.py'"'"'; __file__='"'"'/tmp/pip-wheel-oei7st_8/pythontwitchbotframework/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-wheel-oei7st_8/pythontwitchbotframework/pip-egg-info
         cwd: /tmp/pip-wheel-oei7st_8/pythontwitchbotframework/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-wheel-oei7st_8/pythontwitchbotframework/setup.py", line 3, in <module>
        with open('requirements.txt') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by sharkbound** [comment]
i am not able to reproduce this, it seems to only happen on linux
what python version are you using? and what distro?

**Comment by sharkbound** [comment]
also, can you try with this version and let me know if it works?
```
pip3 install PythonTwitchBotFramework==1.11.9
```

**Comment by thatch** [comment]
It only happens when using `--no-binary` to force using the sdist. 1.11.9 is the same as 1.11.4 in this respect.

**Comment by sharkbound** [comment]
merged in d8d3680
added in pypi version `1.11.10`

**Comment by sharkbound** [comment]
also, thank you for the PR
